src/TokenFactory.php

```diff
@@ -57,10 +57,15 @@ public function validate(string $token): array
             throw new InvalidTokenException($e->getMessage(), (int) $e->getCode(), $e);
         }
 
-        if ($claims['aud'] !== $_SERVER['SERVER_NAME']) {
+        if ($claims['aud'] !== $this->getAudience()) {
```

Comment:
Why would `$claims['aud'] !== $this->getAudience())` not always be populated with either Server_name or LDAP here ??

Reply:
Ok -- understood